services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -327,10 +333,19 @@ async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
             containers_inspect
         )
 
-        # TODO: ANE using `were_service_containers_detected_before` together with
-        # how many containers to expect, it can be detected if containers
-        # died and these can be restarted. Best way to go about it is
-        # to have a different handler trigger in this case registered for idling!
+        # NOTE: All containers are expected to be either created or running.
+        # Extra containers (utilities like forward proxies) can also be present here,
```

Comment:
Q: how are these extra containers setup? are these in the docker-compose specs as well?

Reply:
- They will be coming in the future with https://github.com/ITISFoundation/osparc-simcore/pull/3606 
- They will be added in the docker-compose specs.